Uueoa-Esa, Part 4: Untouched Lands

RustingWithYou opened the pull request:
Adds several ruins for the Untouched Lands of Moghes

RustingWithYou commented:
!review

RustingWithYou commented:
!review

FluffyGhoster commented on maps/random_ruins/exoplanets/moghes/moghes_bar.dm:
```diff
@@ -0,0 +1,152 @@
+	id = /obj/item/card/id
+
+/obj/outfit/admin/moghes_bar_inkeeper/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	var/uniform_color = "[pick("#473b13", "#134718", "#0a6959")]"
+	if(H?.w_uniform)
+		H.w_uniform.color = uniform_color
+	if(H?.shoes)
+		H.shoes.color = "#423509"
```
Might aswell return early if no human is passed in as parameter, also useless var

FluffyGhoster commented on maps/random_ruins/exoplanets/moghes/moghes_bar.dm:
```diff
@@ -0,0 +1,152 @@
+	l_ear = null
+	id = /obj/item/card/id
+
+/obj/outfit/admin/moghes_bar_server/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	var/uniform_color = "[pick("#473b13", "#134718", "#0a6959")]"
+	if(H?.w_uniform)
+		H.w_uniform.color = uniform_color
+	if(H?.wear_suit)
+		H.wear_suit.color = "#423509"
+	if(H?.shoes)
+		H.shoes.color = "#423509"
```
As above

FluffyGhoster commented on maps/random_ruins/exoplanets/moghes/moghes_bar.dm:
```diff
@@ -0,0 +1,152 @@
+	id = null
+
+/obj/outfit/admin/moghes_bar_patron/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	var/uniform_color = "[pick("#473b13", "#134718", "#0a6959", "#d2d9d8", "#262928")]"
+	if(H?.w_uniform)
+		H.w_uniform.color = uniform_color
+	if(H?.wear_suit)
+		H.wear_suit.color = "#423509"
+	if(H?.shoes)
+		H.shoes.color = "#423509"
```
As above

FluffyGhoster commented on maps/random_ruins/exoplanets/moghes/moghes_diona_traders.dm:
```diff
@@ -0,0 +1,113 @@
+	r_pocket = /obj/item/storage/wallet
+
+/obj/outfit/admin/moghes_diona/assistant/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(H?.w_uniform)
+		H.w_uniform.color = "#EAEAEA"
+		H.w_uniform.accent_color = "#EAEAEA"
+	if(H?.wear_suit)
+		H.wear_suit.color = "#42330f"
```
As above minus the var

FluffyGhoster commented on maps/random_ruins/exoplanets/moghes/moghes_heph_mining.dm:
```diff
@@ -0,0 +1,88 @@
+	backpack_contents = list(/obj/item/storage/wallet/random = 1, /obj/item/reagent_containers/food/snacks/koisbar_clean = 3)
+
+/obj/outfit/admin/moghes_heph_miner/klax/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(H?.wear_mask && H.species.has_organ[BP_PHORON_RESERVE])
+		var/obj/item/organ/internal/vaurca/preserve/preserve = H.internal_organs_by_name[BP_PHORON_RESERVE]
+		H.internal = preserve
+		H.internals.icon_state = "internal1"
+	if(H?.wear_suit)
+		H.wear_suit.color = "#2a2b2e"
+	var/obj/item/organ/B = new /obj/item/organ/internal/augment/tool/drill(H)
+	var/obj/item/organ/external/affectedB = H.get_organ(B.parent_organ)
+	B.replaced(H, affectedB)
+	H.update_body()
```
Return early if the parameter is not specified

FluffyGhoster commented on maps/random_ruins/exoplanets/moghes/moghes_thakh.dm:
```diff
@@ -0,0 +1,101 @@
+	id = null
+
+/obj/outfit/admin/moghes_thakh/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(H?.wear_suit)
+		H.wear_suit.color = "#423509"
+	if(H?.shoes)
+		H.shoes.color = "#423509"
```
Etc. etc. as above

Eyeveri reviewed (changes requested):
Is this intentional? The left and right pews being inverted.
![image](https://github.com/Aurorastation/Aurora.3/assets/98699252/248a433f-c032-43b5-bb82-b6436d31da32)

All maps look great! Can't find any other reasons to not approve aside from the pews. 

The code looks fine to me on all of this, but again, I'm still learning the code side of this so Dreamy or Fluffy should take another look at them.

RustingWithYou commented:
> Is this intentional? The left and right pews being inverted. ![image](https://private-user-images.githubusercontent.com/98699252/324184492-248a433f-c032-43b5-bb82-b6436d31da32.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM3NDQ5NjMsIm5iZiI6MTcxMzc0NDY2MywicGF0aCI6Ii85ODY5OTI1Mi8zMjQxODQ0OTItMjQ4YTQzM2YtYzAzMi00M2I1LWJiODItYjY0MzZkMzFkYTMyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIyVDAwMTEwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZlYTA4ZjZhYWYyMzg1YmVkNjM1MTJiZWUxNjc5OWY2ZmQ0MTI2NjMyODU0NWZlZTBhZWZiY2RhMjYwODQ4MmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.vLi89YAWkX-SczkwCIK2iTGATBN4WRNk1NyJq3sgmr8)
> 
> All maps look great! Can't find any other reasons to not approve aside from the pews.
> 
> The code looks fine to me on all of this, but again, I'm still learning the code side of this so Dreamy or Fluffy should take another look at them.

Addressed and replaced with actual pews
